x-pack/plugins/security_solution/server/lib/detection_engine/signals/filters/filter_events.ts

```diff
@@ -21,13 +20,17 @@ export const filterEvents = <T>({
   return events.filter((item) => {
     return fieldAndSetTuples
       .map((tuple) => {
-        const eventItem = get(tuple.field, item._source);
-        if (eventItem == null) {
-          return true;
-        } else if (tuple.operator === 'included') {
+        const eventItem = item.fields ? item.fields[tuple.field] : undefined;
```

Comment:
Super nit: wonder if we're gonna be doing this a lot if it's worth just creating a tiny util to extract `x` field from `fields`.

Reply:
Yeah if it becomes a common pattern I'd support pulling it out into a function